vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/Constants.java

```diff
@@ -77,5 +77,8 @@ public class Constants {
     // Application Importer types
     public static final String APP_IMPORTER_STANDALONE_TYPE = "app-importer-standalone-type";
     public static final String APP_IMPORTER_DOT_INPUTS_TYPE = "app-importer-dot-inputs-type";
+    //Execution type
+    public static final String APP_EXECUTION_LOCAL = "file";
+    public static final String APP_EXECUTION_CLOUD = "lfn";
```

Comment:
Good thing to use constants !

`APP_IMPORTER_FILE_PROTOCOL` and `APP_IMPORTER_LFN_PROTOCOL` are better names.